src/os/freertos/CMakeLists.txt

```diff
@@ -12,6 +13,8 @@ set(LIB_HEADERS
     include/BaseTask.h
     include/BaseTask.tpp
 
+    include/ConditionVariable.h
```

Comment:
For safety, you could include this header in the same tests of Task to make sure it can compile